src/pubtools/_marketplacesvm/tasks/community_push/command.py

```diff
@@ -261,9 +261,10 @@ def set_accounts(acct_name: str, acct_dict: Dict[str, List[str]]) -> None:
             log.info("Loading %s from StArMap: %s", acct_name, accts)
             if isinstance(accts, dict):
                 combined_accts = []
-                for _, accounts in accts.items():  # expected format: Dict[str, List[str]]
-                    combined_accts.extend(accounts)
-                    log.debug("Loaded \"%s\": \"%s\" from StArMap.", acct_name, accounts)
+                for _, accounts in accts.items():  # expected format: Dict[str, str]]
+                    if isinstance(accounts, str):
+                        combined_accts.append(accounts)
+                        log.debug("Loaded \"%s\": \"%s\" from StArMap.", acct_name, accounts)
```

Comment:
We have just one log message for ` isinstance(accts, list)`

It woudl be nice to have just one log message here as well (after for loop)

Reply:
Nice catch! I've set it to be a single log as well (after loop)